ndm-operator.yaml

```diff
@@ -121,13 +121,16 @@ spec:
       # Use host network as container network to monitor udev source using netlink
       # to detect disk attach and detach events using fd.
       hostNetwork: true
+      hostPID: true
```

Comment:
@kmova Will it be an issue to use `hostPID` here.
To check iscsi status, it is required.